src/components/modules/ModuleCheckout.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from "react";
+import { View } from "react-native";
+import Placeholder from "rn-placeholder";
+import {
+  IOColors,
+  IOModuleIDPSavedVSpacing,
+  IOModuleStyles,
+  useIOExperimentalDesign
+} from "../../core";
+import { IOLogoPaymentType, LogoPayment } from "../logos";
+import { VSpacer } from "../spacer";
+import { Body, H6 } from "../typography";
+import { PressableModuleBase } from "./PressableModuleBase";
+
+type ModuleCheckoutProps = { ctaText: string } & (
+  | {
+      paymentLogo?: IOLogoPaymentType;
+      header: string;
+      body: string;
+      isLoading?: false;
+      onPress?: () => void;
+    }
+  | {
+      isLoading: true;
+    }
+);
+
+type CtaOnlyProps = { text?: string };
+
+export const ModuleCheckout = (props: ModuleCheckoutProps) => {
+  if (props.isLoading) {
+    return <LoadingVersion text={props.ctaText} />;
+  }
+  return (
+    <PressableModuleBase onPress={props.onPress}>
+      {props.paymentLogo && <LogoPayment name={props.paymentLogo} />}
+      <View>
+        <H6>{props.header}</H6>
+        <Body>{props.body}</Body>
+      </View>
```

Comment:
Same as skeleton component. In this case we use a new spacing variable because `8` makes the two blocks too tight:
```jsx
const paymentLogoEndMargin: IOSpacingScale = 12;
```
```suggestion
      <View style={[IOStyles.row, IOStyles.alignCenter]}>
        {props.paymentLogo && (
          <View style={{ marginEnd: paymentLogoEndMargin }}>
            <LogoPayment name={props.paymentLogo} />
          </View>
        )}
        <View>
          <H6>{props.header}</H6>
          <Body>{props.body}</Body>
        </View>
      </View>
```